### PR TITLE
add SPARROW's answers to the list of use-cases in the community propo…

### PR DESCRIPTION
**Pull request by Pl-Mrcy**
…sal column.

### CODE REVIEW

**Comment by lukwlodarczyk** [comment]
Hi @Pl-Mrcy 
I believe it would be extremely useful for all group members to add here exact links to paragraphs in Sparrow proposal addressing particular use case. Similarly tu Impression and Viewability Measurement use case.

If there is no straightforward reference at least to have redirect to  paragraph where we can understand this from context

Same thing would apply to TD proposal and probably we can add this in later edits.

@benjaminsavage WDYT?

**Comment by michaelkleber** [comment]
I'm concerned about some of the claims of support made here — I can't see how they're possible with what I understood of the SPARROW design.  I've left a few questions in https://github.com/BasileLeparmentier/SPARROW/issues/11.

File: support_for_advertising_use_cases.md [hunk 1]
```diff
@@ -66,21 +66,21 @@ The ads industry has endeavored to find good ways of answering these basic quest
...
+| [Conversion Fraud](#conversion-fraud) | A section of the [Multi-Browser Aggregation Service Explainer](https://github.com/WICG/conversion-measurement-api/blob/master/SERVICE.md#authenticating-inputs) describes using a service that includes an authentication mechanism. This use case is also being explored for the event level API in this [GitHub Issue](https://github.com/WICG/conversion-measurement-api/issues/13). | Confirmation this is a use-case Safari cares about addressing. In the process of collaboratively designing a solution on this [GitHub Issue](https://github.com/WICG/ad-click-attribution/issues/27). | Facebook proposal for “[Private Fraud Prevention](https://github.com/siyengar/private-fraud-prevention)” <br><br> SPARROW : Thanks to exact reporting, it will be possible to link conversion to both an interest group and a publisher. It will also be possible to have user-level granularity on the website, helping to spot fraudulent activity. |
```
Comment (michaelkleber) [inline support_for_advertising_use_cases.md:77]: > SPARROW : Thanks to exact reporting, it will be possible to link conversion to both an interest group and a publisher.

I don't see how this is compatible with the privacy goals of TURTLEDOVE, which the SPARROW explainer says are its goals also.  Your [Privacy Considerations section](https://github.com/BasileLeparmentier/SPARROW#privacy-considerations) says "The Gatekeeper receives interests group x publisher data, but cannot link this data to individual users since it has no user-level information."  Linking to a specific conversion seems to contradict that.

> It will also be possible to have user-level granularity on the website, helping to spot fraudulent activity.

Again this contradicts "The Gatekeeper [...] has no user-level information."  And I'm not sure what "the website" refers to here, but if it's the publisher site then I don't see how to reconcile this with "Web sites cannot learn the interest groups of the people who visit them" — one of the two original privacy goals.

File: support_for_advertising_use_cases.md [hunk 2]
```diff
@@ -177,23 +177,23 @@ These use-cases are more focused on the publisher's perspective.
...
 |[Search](#search) | unclear, but relies a lot on contextual | unclear | |
-| [Ad Safety](#ad-safety) | Very limited support as opaque iframe aggregated reporting allow for very little ex post audit. | No Support |  |
-| [Advertisers exclusion](#advertisers-exclusion) | Unclear. To be defined. |  |  |
-| [Revenue management](#revenue-management) | Little support in TURTLEDOVE, as nor reporting nor updating bidding JavaScripts is not real-time. This means delays in both information and action, leading to very complex and inaccurate budget/revenue management. |  |  |
+| [Ad Quality](#ad-quality) | Very limited support as opaque iframe aggregated reporting allow for very little ex post audit. | No Support | SPARROW: The Gatekeeper is reponsible for enforcing the publisher policy. The Gatekeeper could provide a service to ensure Ad Quality before the ad is displayed. <br> The aggregated report and Delayed served ads report can be leveraged for ex post ad quality audit. |
```
Comment (michaelkleber) [inline support_for_advertising_use_cases.md:194]: > SPARROW: The Gatekeeper is reponsible for enforcing the publisher policy.

This statement worries me.  Publishers don't interact with the Gatekeeper; it's a piece of browser-trusted infrastructure.  Surely publishers interact with their own ad network in setting their brand safety policies, and the publisher's ad network is the one responsible for doing the right thing.

The actual execution of the code might happen in the browser (for TURTLEDOVE) or in the Gatekeeper (for SPARROW).  But _responsibility_ isn't being moved around, only which computer is executing the instructions.

> The Gatekeeper could provide a service to ensure Ad Quality before the ad is displayed.

This worries me even more!  It makes it look like you're assigning deep decision-making to the Gatekeeper.  Again, in TURTLEDOVE we're not changing whose code makes a decision, only where that code runs.  If SPARROW isn't sticking to that model, then your explainer needs some much more explicit discussion of this fact.

File: support_for_advertising_use_cases.md (hunk 1 above)
Comment (Pl-Mrcy) [reply support_for_advertising_use_cases.md:77]: Everything here is advertiser side. 
The advertiser is the one getting conversion information, not the gatekeeper. The advertiser can use the click_id in the granular reporting to join to first-party user-level data gathered on his domain.
I will edit the section to make it clearer.

File: support_for_advertising_use_cases.md (hunk 2 above)
Comment (Pl-Mrcy) [reply support_for_advertising_use_cases.md:194]: > SPARROW: The Gatekeeper is responsible for enforcing the publisher policy.

Let me rephrase: "the publisher expresses its policy to the gatekeeper so that the Gatekeeper can enforce it."

Indeed, the Gatekeeper only acts on behalf of other actors and merely runs their logic. The gatekeeper doesn't take any decision on its own.
The publisher side (publisher / ad-network) is blind on the interest group/targeting domain. Filtering for ad quality purposed has to be delegated to the entity running the JS (browser/gatekeeper. 

This seems aligned with what is explained in [here](https://github.com/BasileLeparmentier/SPARROW/issues/5#issuecomment-628846717):
> Publisher brand safety could work the same way: the metadata about an interest-group-targeted ad includes its topics (as determined by a server, just like today), publisher controls let them pick what topics are allowed/blocked, and on-device JS compares the two.

Is that correct?

> The Gatekeeper could provide a service to ensure Ad Quality before the ad is displayed.

This sentence refers to something that was mentioned during one the W3C calls. Ads could be "blocked by default" by the gatekeeper before being whitelisted by the publisher-side.
It is only a side note, but it will be removed in the next version, as it may indeed bring some confusion.

File: support_for_advertising_use_cases.md (hunk 1 above)
Comment (michaelkleber) [reply support_for_advertising_use_cases.md]: OK sure.  I've opened https://github.com/WICG/sparrow/issues/16 to talk about my privacy worries here.

**Comment by michaelkleber** [comment]
OK, looks good to me.
